### PR TITLE
Implement role base access in api.

### PR DESCRIPTION
**Pull request by shnooga**

### CODE REVIEW

**Review by Roberto-Meza — COMMENTED** [review]
Wouldn't be a good idea to also change all the other antMatchers by mvcMatchers??

**Review by Roberto-Meza — APPROVED** [review]

**Review by AaronHanich — APPROVED** [review]

**Review by jesusgudino — APPROVED** [review]

**Comment by shnooga** [comment]
> Wouldn't be a good idea to also change all the other antMatchers by mvcMatchers??

Done deal.

File: api/src/main/kotlin/edu/wgu/osmt/security/SecurityConfig.kt [hunk 1]
```diff
@@ -23,14 +23,14 @@ import edu.wgu.osmt.RoutePaths.SKILL_UPDATE
 import edu.wgu.osmt.RoutePaths.TASK_DETAIL_BATCH
 import edu.wgu.osmt.RoutePaths.TASK_DETAIL_SKILLS
 import edu.wgu.osmt.RoutePaths.TASK_DETAIL_TEXT
-import edu.wgu.osmt.RoutePaths.createUuidRegex
```
Comment (JohnKallies) [inline api/src/main/kotlin/edu/wgu/osmt/security/SecurityConfig.kt:26]: If we are removing the import here, is the method still used anywhere? I'd say probably not. Can we remove it?

**Review by JohnKallies — APPROVED** [review]
love it!

**Review by AaronHanich — APPROVED** [review]

**Pull request merged**